index.html

```diff
@@ -5,26 +5,32 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="script.js"></script>
     <title>Document</title>
     <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700;900" rel="stylesheet">
 </head>
 
 <body>
     <header>
     
         <nav>
-        <a href="#" class="logo rotate">Gul</a>
-        <a href="#" class="menu-bars"><img src="images/Union.png"></a>
+
+        
+        <a href="#" class="logo">Gul</a>
+        <img src="images/Union.png" onclick="openNav()" class="menu-bars" >
+
         <ul id="desktop-menu">
             <li><a href="#">Portfolio</a></li>
             <li><a href="#">About</a></li>
             <li><a href="#">Contact</a></li>
```

Comment:
- [ ] Looking at the Figma design, you can see that the menu list items are aligned by the left but your design is placed in the center, I'll suggest you update this. :+1: :wave: 
- - Your design:
![Selection_094](https://github.com/umarahgul/Portfolio/assets/107632780/9c784594-0f9f-4d4c-bcbb-39e2dc752b87)

- - Figma:
![Screenshot from 2023-07-24 22-58-55](https://github.com/umarahgul/Portfolio/assets/107632780/fd7aba64-7cac-408e-b630-cf984a17c414)

- [ ] Also looking at your menu, you can see there are spaces at the left, right, and bottom of the website. when the menu is open, it's meant to cover the entire screen and it should be fixed also i.e. scrolling the website, the menu should still remain in view until it is closed. I'll suggest you work on this too. :smile: :small_airplane: :waffle:  

- - Screenshot:
![Selection_095](https://github.com/umarahgul/Portfolio/assets/107632780/d2cffaca-94a3-4ce1-87bd-aa16610e9315)